docs/integrations-in-rancher/istio/istio.md

```diff
@@ -71,6 +75,10 @@ To remove Istio components from a cluster, namespace, or workload, refer to the
 
 > By default, only cluster-admins have access to Kiali. For instructions on how to allow admin, edit or views roles to access them, see [this section.](rbac-for-istio.md)
 
+:::note
+For Istio installations version `103.1.0+up1.19.6` and later, Kiali uses a token value for its authentication strategy. The name of the Kiali service account in Rancher is `kiali`. Use this name if you are writing commands that require you to enter the name of the Kiali service account (for example, if you are trying to generate or retrieve a session token). For more information, refer to the [Kiali token authentication FAQ](https://kiali.io/docs/faq/authentication/).
```

Comment:
Since we have the same text in two places on the page, maybe it should go in the parent section.

Reply:
I'm not sure the note would fit that well into the parent section as it's more of an overview of all Istio related tooling but I think the note being in the "Kiali" and "Accessing Visualizations" section is a better placement for users that would run into needing to generate a token. Let me know what you think, thanks for the review Marty!

Reply:
I guess in this case it makes sense to keep it closer to where the instructions are